config/feed.php

```diff
@@ -16,7 +16,7 @@
 			/*
 			 * The feed will be available on this url.
 			 */
-			'url' => '/feed',
+			'url' => '',
```

Comment:
What is the purpose of this change? What are the impacts of changing this from `/feed` to the empty URL?

Reply:
This is due to Spacie fucking up... See here: https://github.com/spatie/laravel-feed/issues/151

Reply:
I would advocate to change the comment in the source code above this line and add a few words to the comment telling the reader

 1. that leaving this configuration value empty is intentional,
 2. that the library adds `/feed` by itself and
 3. link to the issue which @ildyria pointed to